fk/teamfetch.go

```diff
@@ -55,13 +55,14 @@ func teamFetch() exitCode {
 			continue
 		}
 
-		if err := fetchAndUpdateRoster(membership.Team, unlockedKey); err != nil {
+		updatedTeam, err := fetchAndUpdateRoster(membership.Team, unlockedKey)
+		if err != nil {
 			out.Print(ui.FormatWarning("Failed to check team for updates", []string{}, err))
 			sawError = true
 			continue
```

Comment:
oof, just because we failed to update the team doesn't mean we shouldn't carry on and fetch our team's keys